CHANGES.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
-- Remove unused values to help ci(@bikallem #873)
+- Address warning 32 errors (@bikallem #876)
+- Remove unused values. (@bikallem #873)
```

Comment:
Please remove these two, and instead modify line 14/15 into:
```
- Various internal refactors (rgrinberg, mseri, bikallem #876, #873, #820, #800, #799, #797)
```

Reply:
Up to you if you want to use the `@`, I had started removing them since a number of people were annoyed by the continuous emails from github due to CHANGELOG mentions

Reply:
Done.